src/io.jl

```diff
@@ -71,6 +71,36 @@ function close(file::FileHandle)
     return nothing
 end
 
+# Info
+"""
+    MPI.File.get_info(file::FileHandle)
+
+Returns the hints for a file that are actually being used by MPI.
+
+# External links
+$(_doc_external("MPI_File_get_info"))
+"""
+function get_info(file::FileHandle)
+    file_info = Info(init=false)
+    @mpichk ccall((:MPI_File_get_info, libmpi), Cint,
+        (MPI_File, Ptr{MPI_Info}), file, file_info)
```

Comment:
will also need to attach finalizer and increment reference counter.

Reply:
ok done